coop_point_of_sale/views/view_pos_session.xml

```diff
@@ -31,6 +31,11 @@
             <field name="start_at" position="before">
                 <field name="cycle"/>
             </field>
+            <field name="stop_at" position="after">
+                <field name="order_qty"/>
+                <field name="amount_subtotal"/>
+                <field name="total_amount"/>
```

Comment:
```suggestion
                <field name="order_count" sum="Count"/>
                <field name="amount_untaxed" sum="Total Untaxed"/>
                <field name="amount_total" sum="Total"/>
```

Besides renaming the fields to the suggested names, you need `sum=""` if you want to show the totals in the grouped tree view! That was in the requirement